### ISSUE TITLE
Upgrade to Wiki.js 2.0.0-beta.275 issues

### ISSUE THREAD

**Issue by devp01** [Requarks/wiki#1003]
**Question**
Describe in details what you are trying to achieve and where do you need guidance.

**Host Info (please complete the following information):**
OS: [Ubuntu 18.04]
Wiki.js version: [Wiki.js 2.0.0-beta.241]
Database engine: [mysql ]

**Comment by devp01** [comment]
After the upgrade from Wiki.js 2.0.0-beta.241 to Wiki.js 2.0.0-beta.275 the site is not getting loaded. But I can see that the repo folder is sync with the git repository. 
error loading the css
The result is a blank page instead of the home page.
![image](https://user-images.githubusercontent.com/46513658/64021490-b7ec0a80-cb51-11e9-9c3a-f2d0603ac83d.png)

**Comment by NGPixel** [comment]
Please post the full contents of the console, not just this tiny screenshot.

**Comment by devp01** [comment]
Hi NGPixel,
An update from me regarding this issue.
The problem was to do with the fact that I have created a page with iframe and was linking one of the Navmenu items to this page which was causing the issue. Removed the page from the Nav menu, so I was able to access the Wiki home page. Logged in to the admin panel and disabled the Block iframe section on it. So it seems to be working fine now. Thanks.
